paywall/src/components/content/CheckoutContent.tsx

```diff
@@ -69,12 +92,22 @@ export default function CheckoutContent() {
     allowClosingCheckout = true
   }
   // hide the checkout iframe
-  const hideCheckout = () => {
+  const hideCheckout = useCallback(() => {
```

Comment:
eslint auto-adds this on commit, so I guess we need it! This memoizes the hideCheckout callback so we don't get a new one on every render. Because this just posts a message to the main window, there's no harm in not memoizing, which is why I hadn't done it before.